scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/SchedulerEvent.java

```diff
@@ -72,6 +72,8 @@
     TASK_PENDING_TO_RUNNING("Task pending to running"), /** A task has just finished. */
     TASK_RUNNING_TO_FINISHED(
             "Task running to finished"), /** A task has just had an error, it will wait for restart. */
+    TASK_IN_ERROR_TO_FINISHED(
```

Comment:
The tests complain, we need to change the following test class: org.ow2.proactive.scheduler.common.TestSchedulerEvent.testSchedulerEvent

It expects 15 events but we added one so we need to check for 16 events